news/163-cache-run-exports

```diff
@@ -0,0 +1,20 @@
+### Enhancements
+
+* <news item>
+
+### Bug fixes
+
+* Move ``run_exports.json`` query into cache, instead of directly using SQL in
+  ``ChannelIndex``. (#163)
```

Comment:
```suggestion
* Move `run_exports.json` query into cache, instead of directly using SQL in
  `ChannelIndex`. (#163)
```

Reply:
I can never remember which are markdown and which are rst